examples/placeholder/linux/include/TestCommand.h

```diff
@@ -74,7 +74,6 @@ class TestCommand
         {
         case chip::DeviceLayer::DeviceEventType::kCommissioningComplete:
             ChipLogProgress(chipTool, "Commissioning complete");
-            chip::DeviceLayer::PlatformMgr().RemoveEventHandler(OnPlatformEvent, arg);
```

Comment:
How about moving this at the end (before the break) instead of removing it?

It seems  like the path is:
src/include/platform/internal/GenericPlatformManagerImpl.cpp does a:
  chip::Platform::MemoryFree(eventHandler);

which I guess kills the TestCommand.

Was this removal always wrong or was it just misplaced?

Reply:
Looks like it may have been misplaced. I updated as you suggested and it no longer crashes. Thank you!